Formula/cli53.rb

```diff
@@ -0,0 +1,22 @@
+class Cli53 < Formula
+  desc "command-line tool for Amazon Route 53"
+  homepage "https://github.com/barnybug/cli53"
+  url "https://github.com/barnybug/cli53/archive/0.7.4.tar.gz"
+  sha256 "3cb89e6aa91676ffd0577798a4b06b056667d18ad836de4fa31c0564ee48474e"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/barnybug"
+    ln_s buildpath, buildpath/"src/github.com/barnybug/cli53"
+
+    system "make", "build"
+
+    bin.install "cli53"
+  end
+
+  test do
+    system "#{bin}/cli53", "--version"
```

Comment:
Tweaked it slightly in [post](https://github.com/Homebrew/homebrew-core/commit/e1a7145e900a67a53de5f6f65a66e366cd6c451b#diff-9ca55149f206b01dbb5e693fbd58b57dR19) but yeah, there's not much we can do here on functionality given the credential issue. There's a few formulae like this, so it's not unheard of.